test/CompareComply.v1.UnitTests/CompareComplyV1UnitTests.cs

```diff
@@ -222,6 +223,50 @@ public void ConvertToHtml_Success()

```

Comment:
Is there support in C# for loading in JSON test resources? I'm thinking maintenance-wise it would be less of a headache than messing with an inline JSON string.